wp-content/themes/thestanforddaily/functions.php

```diff
@@ -149,6 +149,29 @@ function tsd_scripts() {
 }
 add_action( 'wp_enqueue_scripts', 'tsd_scripts' );
 
+/**
```

Comment:
If you have a source for this (e.g. http://www.transformationpowertools.com/wordpress/automatically-shorten-manual-excerpt / https://wordpress.stackexchange.com/a/76039/75147), it is always better to also add the link to the source in the comments so that we can refer to the original code in the future.